config/reports/berichten/messages-with-provenance.js

```diff
@@ -42,6 +42,8 @@ async function generate() {
     const recipient = detailsStore.getObjects(message, ns.sch`recipient`)[0];
     const recipientV = recipient?.value;
     const recipientnameV = detailsStore.getObjects(recipient, ns.skos`prefLabel`)[0]?.value;
+    const recipientClassificatieCode = detailsStore.getObjects(recipientV, ns.besluit`classificatie`)[0]?.value;
```

Comment:
I have an issue with that, loging `recipientClassificatieCode` will be undefined but `recipientTypeBestuur` will sometimes hold the right value and sometimes the wrong one, I'm not sure why and I'm a bit of a noob with N3.js

I got a sample of the QA data from the report.

```
conversation;identifier;about;message;sender;sendername;recipient;recipientname;recipientTypeBestuur;type;dateSent;dateReceived;provenance;attachmentSequence;filename;content
http://data.lblod.info/id/conversaties/d1eadd00-e38d-11ed-9376-0242c0a81019;PerfTest_Bulk-25-04;testing bulk verzending performance;https://kalliope-test.abb.vlaanderen.be/#!/case/detail/f87283f6-6df0-4578-9f26-cdb77534e967/33b81240-4c86-4c1f-b8bc-ca7db3cf3ab3;http://data.lblod.info/id/bestuurseenheden/141d9d6b-54af-4d17-b313-8d1c30bc3f5b;Agentschap Binnenlands Bestuur;http://data.lblod.info/id/bestuurseenheden/319db6e275f281d0280da90d6f6aba3462f4e47b6f53a34639feb91015a5822b;Oud-Heverlee;Oud-Heverlee;Opvraging;2023-05-05T03:42:36+02:00;2023-05-05T03:59:10+02:00;https://github.com/lblod/berichtencentrum-sync-with-kalliope-service;1/1;handleiding_post_raadplegen_v3_1.pptx;testing performance of bulk sending 300 post UIT
http://data.lblod.info/id/conversaties/85517604-e386-11ed-9376-0242c0a81019;PerfTest_Bulk-25-04;testing bulk verzending performance;https://kalliope-test.abb.vlaanderen.be/#!/case/detail/f87283f6-6df0-4578-9f26-cdb77534e967/3246447f-d65e-4031-8d06-79097cc84831;http://data.lblod.info/id/bestuurseenheden/141d9d6b-54af-4d17-b313-8d1c30bc3f5b;Agentschap Binnenlands Bestuur;http://data.lblod.info/id/bestuurseenheden/8df96cc548c53410332620ec1adae4591bd5340127b1332c4b902c5c3afe260d;Oosterzele;Gemeente;Opvraging;2023-05-05T04:14:19+02:00;2023-05-05T04:34:19+02:00;https://github.com/lblod/berichtencentrum-sync-with-kalliope-service;1/1;2019 GR notulen 3jan.pdf;testing performance of bulk sending 300 post UIT
http://data.lblod.info/id/conversaties/39bb312e-eaa4-11ed-bb5d-0242c0a81019;PerfTest_Bulk-25-04;testing bulk verzending performance;https://kalliope-test.abb.vlaanderen.be/#!/case/detail/f87283f6-6df0-4578-9f26-cdb77534e967/2f4dc9ab-fec6-4de4-88e9-2e84366f660a;http://data.lblod.info/id/bestuurseenheden/141d9d6b-54af-4d17-b313-8d1c30bc3f5b;Agentschap Binnenlands Bestuur;http://data.lblod.info/id/bestuurseenheden/dfd76f1550853dc2b4bc8d763c7b378c8c8c81588dd4ba3e0ef05cfd0a456927;Herstappe;Gemeente;Opvraging;2023-05-05T04:55:53+02:00;2023-05-05T05:31:29+02:00;https://github.com/lblod/berichtencentrum-sync-with-kalliope-service;1/1;Lorem ipsum dolor sit amet.docx;testing performance of bulk sending 300 post UIT
```

Reply:
It seems my construct query was the issue